cirq/ops/common_gates.py

```diff
@@ -666,6 +685,11 @@ def _circuit_diagram_info_(self, args: 'cirq.CircuitDiagramInfoArgs'
             wire_symbols=('H',),
             exponent=self._diagram_exponent(args))
 
+    def _has_stabilizer_effect_(self) -> Optional[bool]:
```

Comment:
I noticed that it only said we added this because of the addition of the quil function. I tried moving this below the `_quil_` function to see if that will remove the change.